test/ruby.test.js

```diff
@@ -155,7 +155,7 @@ eachConfig((prettierConfig, rubocopConfig, config) => {
 
     eachError(config, (file, getContents) => {
       test(`${file} throws error on parsing`, () => {
-        expect(getContents).toThrowError();
+        expect(getContents).toThrowError('Invalid ruby');
```

Comment:
There were other errors happening under the hood here, but they were a false positive.

I could maybe throw up a PR for adding linting to catch this again in the future?

https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/require-tothrow-message.md